doc/cephfs/cephfs-shell.rst

```diff
@@ -441,3 +441,51 @@ Usage :
 
 Options:
   -h, --help   Shows the help message
+
+df
+--
+
+Display amount of available disk space
+
+Usage :
+
+    df [-h] [file [file ...]]
+
+* file - name of the file
+
+Options:
+  -h, --help   Shows the help message
+
+du
+--
+
+Show disk usage of a directory
+
+Usage :
+
+    du [-h] [-r] [paths [paths ...]]
+
+* paths - name of the directory
+
+Options:
+  -h, --help   Shows the help message
```

Comment:
An option got missed - `  -r          Recursive Disk usage of all directories.`.

Reply:
Done.